cmd/aspect/info/info.go

```diff
@@ -18,7 +19,7 @@ func NewDefaultInfoCmd() *cobra.Command {
 }
 
 func NewInfoCmd(streams ioutils.Streams) *cobra.Command {
-	v := info.New(streams)
+	i := info.New(streams)
```

Comment:
Single-letter variables are tricky to use. The rules are implicit and they can usually lead to a lack of intelligibility of the code.

E.g. `i` is well-known to represent `index` in any language. Perhaps we should stop using single-letter vars for these commands as the scope of the `New<name>Cmd` functions have been increasing recently.

Reply:
That's a good point. Ok, I'll refactor all of these.